src/server/sim-files.js

```diff
@@ -139,7 +139,12 @@ SimulationFiles.prototype._createHostJsFile = function (simulationFilePath, host
 
     var scriptDefs = createScriptDefs(hostType, scriptTypes);
 
-    var b = browserify({ paths: getBrowserifySearchPaths(hostType), debug: true });
+    var b = browserify({
+        paths: getBrowserifySearchPaths(hostType),
+        debug: true,
+        externalRequireName: '_cordovaSimulateRequire'
```

Comment:
Yeah, that's what I'm wondering - if the proper fix is to just explicitly set `hasExports` to `false` (or whatever we need to do to make it `false`).